terraso_backend/config/settings.py

```diff
@@ -232,6 +232,11 @@
 
 MICROSOFT_CLIENT_ID = config("MICROSOFT_CLIENT_ID", default="")
 MICROSOFT_CLIENT_SECRET = config("MICROSOFT_CLIENT_SECRET", default="")
+MICROSOFT_TENANT = config("MICROSOFT_TENANT", default="")
```

Comment:
```suggestion
```

Reply:
issue: we don't use the tenant anywhere — no need to add it.

Reply:
Yes, this snuck in in a rebase. Thanks for catching it.